plugin/compile-scss.js

```diff
@@ -80,7 +82,7 @@ class SassCompiler extends MultiFileCachingCompiler {
       return result && {
         data: result.css,
         sourceMap: result.sourceMap,
-        };
+      };
```

Comment:
Avoid changing coding style in unrelated places if possible (even if the change of whitespace is correct). This adds noise to the PR and makes review harder.

Reply:
I'm moving it to a separate PR